set-link/entrypoint.sh

```diff
@@ -102,6 +115,9 @@ if [[ -n $story ]]; then
 fi
 
 echo "Final new title is '${new_title}'"
+echo "Final new body is '${new_body}'"
+
+[[ ${IS_TESTING:-} == "true" ]] && exit 0
```

Comment:
Yea, I think this org might be the wrong place for that then? Maybe it needs to be moved to our private org.  That's probably a hassle for wherever it's installed though :( 

Can the test info not be sterilized?

Reply:
> Can the test info not be sterilized?

Possibly, but might be harder, because we rely on the GITHUB_EVENT_DATA json, I wrote script that pulls the JSON from the logs for commit. If in the future we might need additional field from the commit, we can just update the script and rerun it, it'll update all the test JSON files.

Reply:
Oh, looks like [we can move it to the main org, and change the visibility to internal](https://github.com/github/roadmap/issues/74).